staging/src/k8s.io/sample-controller/README.md

```diff
@@ -79,17 +79,44 @@ type User struct {
 
 To validate custom resources, use the [`CustomResourceValidation`](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation) feature.
 
-This feature is beta and enabled by default in v1.9. If you are using v1.8, enable the feature using
```

Comment:
We use `apps/v1` for deployments which means we require a v1.9 cluster anyway. This means CRD validation is enabled by default so we don't need to include feature gate information.